frontend/src/components/NodeChart.vue

```diff
@@ -2,20 +2,12 @@
   <v-container class="overflow-auto">
     <v-row>
       <v-col xs="12" lg="9">
-        <v-sheet
-          :min-height="lgAndUp ? '65vh' : '50vh'"
-          :max-height="lgAndUp ? '100%' : '20vh'"
-          max-width="100%"
-          min-width="500px"
-        >
+        <v-sheet :min-height="lgAndUp ? '65vh' : '50vh'" :max-height="lgAndUp ? '100%' : '20vh'" max-width="100%"
+          min-width="500px">
```

Comment:
this template has a bunch of changes that look like they are formatting only modifications
it looks like maybe you are using some odd eslint settings?
the formatting makes it a bit difficult to see what your actual changes are here.
could you back out the formatting changes so that we get a true diff?
or run eslint on this file?

Reply:
I didn't realize this happened, apologies. I think it was caused by a vue plugin that I installed (vue-language-server). I tried to undo the linting, hopefully it's better now.

Reply:
no worries it's mostly my bad for not setting up linting enforcement from the get-go
we will add that and do a full re-lint at some point